Source/PropertyPolicy.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  PropertyPolicy.swift
+//  KakapoExample
+//
+//  Created by Alex Manzella on 31/03/16.
+//  Copyright © 2016 devlucky. All rights reserved.
+//
+
+import Foundation
+
+public protocol _PropertyPolicy: Serializable {
+    var _object: Any { get }
+    var shouldSerialize: Bool { get }
+}
+
+public protocol PropertyPolicy: _PropertyPolicy {
+    typealias T
+    var object: T? { get }
+    var shouldSerialize: Bool { get }
+}
+
+extension PropertyPolicy {
+    public var _object: Any {
+        get {
+            return object
+        }
+    }
+}
```

Comment:
Just out of curiosity: how would a property policy that wants to, for instance, serialize nil as 0 work? The object proerty will have to return 0 or 1 based on his value?

I think the documentation on how to create custom PropertPolicty is quite crucial, otherwise is a bit  confusing.